FirebaseAI/Sources/GenerativeModel.swift

```diff
@@ -137,6 +137,12 @@ public final class GenerativeModel: Sendable {
     return try await generateContent([ModelContent(parts: parts)])
   }
 
+  public func generateContent(@ModelContentBuilder _ contentBuilder: ()
```

Comment:
Are there any scenarios where this will make it ambiguous which method will get called?